src/Resources/config/twig_hooks/shop/wishlist/index.yaml

```
@@ -0,0 +1,83 @@
+sylius_twig_hooks:
+    hooks:
+        'bitbag_sylius_wishlist_plugin.wishlist.index':
+            content:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content.html.twig"
+                priority: 0
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content':
+            header:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/header.html.twig"
+                priority: 100
+            form:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form.html.twig"
+                priority: 90
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content.form':
+            form:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections.html.twig"
+                priority: 0
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content.form.sections':
+            items:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items.html.twig"
+                priority: 100
+            actions:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/actions.html.twig"
+                priority: 90
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content.form.sections.actions':
+            collective_actions:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/actions/collectiveActions.html.twig"
+                priority: 100
+            global_actions:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/actions/globalActions.html.twig"
+                priority: 90
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content.form.sections.items':
+            head:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head.html.twig"
+                priority: 100
+            body:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/body.html.twig"
+                priority: 90
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content.form.sections.items.head':
+            select_all:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head/selectAll.html.twig"
```

Use snake_case notation for templates name
```suggestion
                template: "@BitBagSyliusWishlistPlugin/wishlist/index/content/form/sections/items/head/select_all.html.twig"
```